### PR TITLE
Feature/error component

### PR DESCRIPTION
**Pull request by e-spitz**
# Description

Add error component for bad url

## Type of change
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Localhost
- [] Console Log
- [ ] Passing Tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -12,18 +13,13 @@ function App() {
     setFavs([favObj, ...favs])
   }
 
-  // const removeFavorite = (id) => {
```
Comment (rachelJensen) [inline src/components/App/App.js:15]: It looks like you have this function operating in another component so you can remove this commented out code

File: src/components/App/App.js [hunk 2]
```diff
@@ -12,18 +13,13 @@ function App() {
...
-  //   console.log(remainingCards)
-  //   setFavs(remainingCards)
-  // }
-
   return (
     <div className="App">
       <Header />
       <Switch>
-        <Route exact path='/' render={() => <GenerateSection addFavorite={addFavorite} favorites={favs}/>}/>
+        <Route exact path='/yuletide-yeezy' render={() => <GenerateSection addFavorite={addFavorite} favorites={favs}/>}/>
```
Comment (rachelJensen) [inline src/components/App/App.js:20]: routes look great

File: src/components/Card/Card.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ export const Card = ({ card, remove }) => {
           <p className='card-quote'>{card.quote}</p>
         </div>
         <img src={card.img}/>
+        <div className='share-btns'>
```
Comment (rachelJensen) [inline src/components/Card/Card.js:12]: Is this share button being used? If not, maybe remove it for now.

**Review by rachelJensen — APPROVED** [review]
Nice job on the routes. This looks really good.

File: src/components/Card/Card.js (hunk 3 above)
Comment (e-spitz) [reply src/components/Card/Card.js:12]: Good eye. Thank you!

**Pull request merged**